libr/reg/profile.c

```diff
@@ -248,7 +249,10 @@ R_API bool r_reg_set_profile(RReg *reg, const char *profile) {
 		eprintf ("r_reg_set_profile: Cannot find '%s'\n", profile);
 		return false;
 	}
-	bool ret = r_reg_set_profile_string (reg, str);
+	r_strbuf_init (&sb);
+	r_strbuf_set (&sb, str);
+	bool ret = r_reg_set_profile_string (reg, &sb);
```

Comment:
i dont see the beneffit of passing a strbuf to set the regprofile instead of just passing the string. you can't compare by pointers here, so its an extra workload that is unnecessary here